gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/schema.yml

```diff
@@ -28,4 +28,16 @@ databaseChangeLog:
               - column:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.